test/functional/mining_mainnet.py

```diff
@@ -63,6 +64,9 @@ def mine(self, height, prev_hash, blocks, node, fees=0):
         block.nBits = DIFF_1_N_BITS
         block.nNonce = blocks['nonces'][height - 1]
         block.vtx = [create_coinbase(height=height, script_pubkey=bytes.fromhex(COINBASE_SCRIPT_PUBKEY), retarget_period=2016)]
+        block.vtx[0].nLockTime = 0
+        block.vtx[0].vin[0].nSequence = SEQUENCE_FINAL
+        block.vtx[0].rehash()
```

Comment:
Just a comment, but I also don't think it would be worth it to re-generate this alternative mainnet chain.

Reply:
Would be good to add a comment here to explain.

We can always regenerate it later.

Reply:
Added a comment.